youtube_dl/extractor/indavideo.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class IndavideoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?indavideo\.hu/video/(?P<id>.+)'
+    _TESTS = [
+        {
+            'url': 'http://indavideo.hu/video/Cicatanc',
+            'md5': 'c8a507a1c7410685f83a06eaeeaafeab',
+            'info_dict': {
+                'id': '1837039',
+                'title': 'Cicatánc',
+                'url': 'http://main-z-director.indavideo.hu/indavideo/911/001/837/039/55c85ad6075237.24088793.360.mp4',
+                'ext': 'mp4',
+                'display_id': 'Cicatanc',
+                'thumbnails': [
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/1-1.jpg', 'id': '0'},
```

Comment:
URLs must contain scheme name. There is no need to thoroughly test all the thumbnails.